v2/components/tooltip/tooltip.tsx

```diff
@@ -33,11 +33,11 @@ export const useHover = (): [React.MutableRefObject<any>, boolean] => {
 export const Tooltip = (props: {content: React.ReactNode | string; inverted?: boolean} & React.PropsWithRef<any>) => {
     const [tooltip, showTooltip] = useHover();
     return (
-        <div style={{position: 'relative', minWidth: 0}}>
+        <div ref={tooltip} style={{position: 'relative', minWidth: 0}}>
```

Comment:
scooted the ref up to the parent, which contains both the tooltip content and the trigger element